### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by lucasvanhalst**

### CODE REVIEW

File: rogerthat/src/main/res/values-nl/allstr.xml [hunk 1]
```diff
@@ -560,24 +554,25 @@
...
-    <string name="rogerthat">OK</string>
     <string name="rogerthat_system">Rogerthat Systeem</string>
     <string name="rogerthat_welcome_enterprise">Welkom bij de %1$s app. \n\nVoordat u van start kunt moet u eerst uw account registreren.</string>
     <string name="sample_news_message_1">Gratis onze huisaperitief voor u en uw gezelschap bij een gezellig dineetje van lekkere tapas!\n\nDeze actie is geldig tot eind januari 2017</string>
     <string name="sample_news_message_2">Wij wensen u prettige feesten en bieden u een KERSTGESCHENK aan ter waarde van 5 euro.\n\nBij aankoop voor een bedrag van minimum 25 euro ontvangt u 5€ directe éénmalige korting.\n\nUit dank voor het jarenlange vertrouwen. \nActie geldig van 17/12/2016 tem 31/01/2017</string>
     <string name="sample_news_title_1">Scan deze coupon bij La Tapa Canaria en ontvang ...</string>
     <string name="sample_news_title_2">KERSTGESCHENK van 5€</string>
     <string name="saturday">Zaterdag</string>
-    <string name="save">Bewaren</string>
+    <string name="save">Opslaan</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-nl/allstr.xml:567]: check

File: rogerthat/src/main/res/values-nl/allstr.xml [hunk 2]
```diff
@@ -560,24 +554,25 @@
...
     <string name="save_canned_message_dialog_title">Geef een naam voor het standaard bericht</string>
     <string name="save_canned_message_no_name">zonder naam</string>
+    <string name="save_job_profile">Jobprofiel opslaan</string>
     <string name="save_profile">Bewaar uw profiel</string>
-    <string name="saved">Opgeslagen</string>
+    <string name="saved">Bewaard</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-nl/allstr.xml:572]: check

File: rogerthat/src/main/res/values-nl/allstr.xml [hunk 3]
```diff
@@ -702,14 +701,16 @@
     <string name="validate_location">Controleer locatie</string>
     <string name="validation_failed">Ongeldige input</string>
     <string name="via">Via</string>
+    <string name="via_x">Via %1$s</string>
     <string name="video">Video</string>
     <string name="view_my_profile">Mijn profiel bekijken</string>
+    <string name="view_saved_jobs">Bekijk bewaarde jobaanbiedingen</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-nl/allstr.xml:707]: check

File: rogerthat/src/main/res/values-ro/allstr.xml [hunk 4]
```diff
@@ -2,7 +2,6 @@
 <resources>
     <string name="__me_as_recipient">eu</string>
     <string name="__me_as_sender">eu</string>
-    <string name="about">Despre</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-ro/allstr.xml:5]: only deleted translations?

File: rogerthat/src/main/res/values-ro/allstr.xml [hunk 5]
```diff
@@ -13,63 +12,43 @@
     <string name="action_scheduled">Hmmm, unele intarzieri in retea. Ne vom intoarce la tine in scurt timp.</string>
     <string name="action_tel">tel</string>
     <string name="action_www">www</string>
-    <string name="actions">Acțiuni</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-ro/allstr.xml:16]: same a lot of removed once

File: rogerthat/src/main/res/values-ru/allstr.xml [hunk 6]
```diff
@@ -1,118 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="__me_as_recipient">я</string>
-    <string name="about">О программе</string>
```
Comment (RubenMattan) [inline rogerthat/src/main/res/values-ru/allstr.xml:4]: same a lot of removed once

File: rogerthat/src/main/res/values-ro/allstr.xml (hunk 4 above)
Comment (lucasvanhalst) [reply rogerthat/src/main/res/values-ro/allstr.xml:5]: well I don't speak romanian so

Comment (lucasvanhalst) [reply rogerthat/src/main/res/values-ro/allstr.xml:5]: In fact none of the apps are build with these languages, only en, nl and fr are used the rest is just thrown away so we might as well remove these files

**Pull request merged**